apps/server/src/modules/health/health.entities.ts

```diff
@@ -0,0 +1,6 @@
+import { AnyEntity } from '@mikro-orm/core';
+import { EntityName } from '@mikro-orm/nestjs/typings';
+
+import { HealthcheckEntity } from './repo/entity';
+
+export const HealthEntities: EntityName<AnyEntity>[] = [HealthcheckEntity];
```

Comment:
```suggestion
import { HealthCheckEntity } from './repo/entity';

export const HealthEntities: EntityName<AnyEntity>[] = [HealthCheckEntity];
```

Reply:
Done in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4191/commits/9f7cb45820e06668af950204e05f82983c112e54 .